xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-tree/xwiki-platform-index-tree-api/src/main/resources/org/xwiki/index/tree/internal/nestedpages/query/queries.hbm.xml.vm

```diff
@@ -114,7 +114,7 @@
 
     ) xwikiPage
 
-    order by lower(xwikiPage.title) $order, xwikiPage.title $order
+    order by BEFORE_ORDER_MODIFIER xwikiPage.title AFTER_ORDER_MODIFIER $order, xwikiPage.title $order
```

Comment:
I don't know if it's possible but might worth adding a comment to indicate the BEFORE/AFTER are replaced in QueryRegistrationHandler to handle collation? That could be a bit surprising to see those when reading the query in a while...